src/providers/monsterapi/generate.ts

```diff
@@ -0,0 +1,118 @@
+// generate.ts
+import { MONSTERAPI } from '../../globals';
+import {
+  ChatCompletionResponse,
+  ErrorResponse,
+  ProviderConfig,
+} from '../types';
+import {
+  generateErrorResponse,
+  generateInvalidProviderResponseError,
+} from '../utils';
+
+export const MonsterAPIChatCompleteConfig: ProviderConfig = {
```

Comment:
As per the API docs, some of the mappings are missing here. Can you please check this once?
If any param is similar to an existing OpenAI param, you can use the OpenAI param name as input and map it to Monster's param. If there is no OpenAI compatible param, then you can add them as a new mapping as it is.